src/Utilities/Compiler/Extensions/ReportDiagnosticExtensions.cs

```diff
@@ -39,14 +39,11 @@ public static bool IsLessSevereThan(this ReportDiagnostic current, ReportDiagnos
                     return false;
 
                 case ReportDiagnostic.Warn:
-                    switch (other)
+                    return other switch
```

Comment:
Isn't it better to use a ternary operator here?

Reply:
Possibly, though I just applied the suggestions that were presented by the IDE rules. I can clean this up in a follow-up change for the whole repo.